codex-rs/tui/src/lib.rs

```diff
@@ -25,13 +25,20 @@ mod bottom_pane;
 mod chatwidget;
 mod citation_regex;
 mod cli;
+#[cfg(feature = "vt100-tests")]
```

Comment:
Is this less work to just always make it `pub`? This is only within our own workspace, anyway.